README.md

```diff
@@ -224,13 +224,23 @@ or using shadow builds.
 
 ## Building for Sailfish SDK
 
-1. create set of directories like '~/QtCreator/Build' '~/QtCreator/Install'
-2. navigate to '~/QtCreator' and clone the Qt Creator's source tree there
-3. run qmake:
-	qmake ../digia-qt-creator/qtcreator.pro -r -after "DEFINES+=IDE_COPY_SETTINGS_FROM_VARIANT=. IDE_SETTINGSVARIANT=SailfishBeta6" QTC_PREFIX=
+Building Qt Creator for the Sailfish SDK is done using some helper scripts,
```

Comment:
```suggestion
Building Qt Creator for the Sailfish SDK is done using tools,
```

The term scripts have some negative connotations :)

Reply:
changed to tools